Examples/ByFeature/DependencyInjection/paket.references

```diff
@@ -1,2 +1,4 @@
 ﻿FSharp.Core
-NUnit
\ No newline at end of file
+NUnit
+Microsoft.NET.Test.Sdk
+NUnit3TestAdapter
```

Comment:
Am I right that you are migrating NUnit2 to NUnit3? I am fine with that but it may make sense to have both in examples (if it does not mean significant additional work). What do you think?

Reply:
Yes, I am migrating to NUnit 3. The main reason is that NUnit 2 neither supports .NET Standard nor it is maintained library - basically it is just older version of the library which is over 4 years old by this time. I could potentially compile it for .NET Framework only and do some special treating in fake build script but I don't see it worth. I wouldn't recommend anybody today to start using TickSpec with NUnit 2 and if somebody has an old NUnit 2 based project it will still work for him the same way.

It makes to me much better sense to make mapping for other current frameworks, e.g. MSTest v2.

Reply:
Sounds good to me.